rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -183,37 +198,6 @@ public static List<UTXO> deserializeUTXOList(byte[] data) {
         return list;
     }
 
-    public static byte[] serializeSet(SortedSet<Sha256Hash> set) {
-        int nhashes = set.size();
-
-        byte[][] bytes = new byte[nhashes][];
-        int n = 0;
-
-        for (Sha256Hash hash : set) {
-            bytes[n++] = RLP.encodeElement(hash.getBytes());
-        }
-
-        return RLP.encodeList(bytes);
-    }
-
-    public static SortedSet<Sha256Hash> deserializeSet(byte[] data) {
-        SortedSet<Sha256Hash> set = new TreeSet<>();
-
-        if (data == null || data.length == 0) {
-            return set;
-        }
-
-        RLPList rlpList = (RLPList)RLP.decode2(data).get(0);
-
-        int nhashes = rlpList.size();
-
-        for (int k = 0; k < nhashes; k++) {
-            set.add(Sha256Hash.wrap(rlpList.get(k).getRLPData()));
-        }
-
-        return set;
-    }
-
```

Comment:
this has been removed since it is not being used any more (it was replaced by serialize/deserialize rsk tx waiting for signatures(